source/reference/regular_expressions.rst

```diff
@@ -21,7 +21,9 @@ syntax described below.
 
 - ``.``: Matches any character except line break characters. (Be aware that what
   constitutes a line break character is somewhat dependent on your platform and
-  language environment, but in practice this rarely matters).
+  language environment, but in practice this rarely matters). To include
+  newlines use the ``(.|\n\r)*`` pattern which avoids the use of regex flags/modifiers
```

Comment:
This should be `\r\n` not `\n\r`, right? And really it should be `\r?\n` to match UNIX style line endings as well as windows style.

Reply:
Is `[\s\S]` not supported by all regex engines?